openapi/components/schemas/Plans/FlexiblePlan.yaml

```
@@ -1,15 +1,36 @@
 type: object
 allOf:
   - $ref: ../Common/CommonPlan.yaml
-  - properties:
-      id:
-        readOnly: false
-        description: The ID of plan to modify with given settings.
-        allOf:
-          - $ref: ../ResourceId.yaml
-  - required:
-    - id
-    - name
-    - currency
-    - productId
-    - pricing
+properties:
+  id:
+    readOnly: false
+    description: The ID of plan to modify with given settings.
+    allOf:
+      - $ref: ../ResourceId.yaml
+  isActive:
+    type: boolean
+    description: Whether a plan is active.
+  invoiceTimeShift:
+    description: You can shift issue time and due time of invoices for this plan.
+    nullable: true
+    allOf:
+      - $ref: ../Invoices/InvoiceTimeShift.yaml
+  description:
+    type: string
+    description: The plan's description.
+  richDescription:
+    type: string
+    description: The plan's description enhanced.
+  _links:
+    type: array
+    description: The links related to resource, including links provided by the list.
```

nitpick: Where possible, we should avoid starting descriptions with "the"

This can be tiresome to read

I'm not sure which list we are referring to, I think we should describe it or remove it

```suggestion
    description: Related resource links.
```